validator/sawtooth_validator/consensus/handlers.py

```diff
@@ -85,10 +83,13 @@ def handle(self, connection_id, message_content):
         except DecodeError:
             response.status = response.BAD_REQUEST
         else:
-            self.handle_request(request, response)
+            handler_status = self.handle_request(request, response)
+
+        if handler_status is None:
```

Comment:
Why not just return this from the `handle_request` implementations?  The the type is always consistently a `HandlerStatus`.  Returning `None` looks confusing, and adds the complexity here.

Reply:
I can make that change.